### PR TITLE
Models creation

### PR DESCRIPTION
**Pull request by cha-fa**
All models are created
Seed is created

### CODE REVIEW

**Comment by cha-fa** [comment]
[Faire la DB](https://trello.com/c/bo3TSKdK/78-template-feature)

File: app/models/user.rb [hunk 1]
```diff
@@ -6,5 +6,15 @@ class User < ApplicationRecord
 
   devise :database_authenticatable, :registerable,
   :jwt_authenticatable, jwt_revocation_strategy: self
+  validates :first_name, presence: true
```
Comment (DanBertrand) [inline app/models/user.rb:9]: A-t-on vraiment besoin du nom et prénom au sign up ?

**Review by DanBertrand — COMMENTED** [review]
Globally good : Just one comment : Do we need first name and last name on the register ?

**Review by DanBertrand — APPROVED** [review]

**Comment by DanBertrand** [comment]
Code reviewed and no issues found (we keep first_name and last_name on the register page)

**Pull request merged**